src/cursor.js

```diff
@@ -326,7 +326,9 @@ class Cursor extends Model {
 
   // Public: Moves the cursor to the bottom of the buffer.
   moveToBottom () {
+    const column = this.goalColumn
     this.setBufferPosition(this.editor.getEofBufferPosition())
+    this.goalColumn = column
```

Comment:
Why is the `goalColumn` only preserved this way in this one method?

Reply:
honestly, there may be. this is the only one that caused a test to fail though.